data_model/src/permission.rs

```diff
@@ -1,220 +1,117 @@
 //! Permission Token and related impls
 #[cfg(not(feature = "std"))]
-use alloc::{
-    collections::{BTreeMap, BTreeSet},
-    format,
-    string::String,
-    vec::Vec,
-};
+use alloc::{collections::BTreeSet, format, string::String, vec::Vec};
 #[cfg(feature = "std")]
-use std::collections::{BTreeMap, BTreeSet};
+use std::collections::BTreeSet;
 
-use derive_more::{Constructor, Display, FromStr};
-use getset::Getters;
-use iroha_data_model_derive::{model, IdEqOrdHash};
+use iroha_data_model_derive::model;
 use iroha_schema::IntoSchema;
 use parity_scale_codec::{Decode, Encode};
 use serde::{Deserialize, Serialize};
-use serde_with::{DeserializeFromStr, SerializeDisplay};
 
 pub use self::model::*;
-use crate::{IdBox, Identifiable, Name, Value, ValueKind};
 
 /// Collection of [`Token`]s
 pub type Permissions = BTreeSet<PermissionToken>;
 
-/// Trait to identify [`ValueKind`] of a type which can be used as a [`Token`] parameter.
-///
-/// On a higher level, all permission token parameters have [`Value`] type, but for now we allow
-/// to define builtin permission tokens with stronger types.
-/// This trait is used to retrieve the [`kind`](`ValueKind`) of a [`Value`] which can be constructed
-/// from given parameter.
-///
-/// Will be removed as well as builtin permission tokens and validators
-/// when *runtime validators* and *runtime permissions* will be properly implemented.
-pub trait ValueTrait: Into<Value> {
-    /// The kind of the [`Value`] which the implementing type can be converted to.
-    const TYPE: ValueKind;
-}
+use super::*;
+
+/// Unique id of [`PermissionToken`]
+pub type PermissionTokenId = String;
```

Comment:
I think we have some forbidden characters than should not be presented in `PermissionTokenId`. I think it should be `Name` at least

Reply:
> I think we have some forbidden characters than should not be presented in PermissionTokenId. I think it should be Name at least

no, this is the name of the struct which comes from `iroha_schema::TypeId::id`

Reply:
check the update code